pandas/core/window/rolling.py

```diff
@@ -1214,27 +1209,15 @@ def count(self):
     """
     )
 
-    def apply(self, func, raw=None, args=(), kwargs={}):
+    def apply(self, func, raw=False, args=(), kwargs={}):
```

Comment:
can this now be annotated?

Doesnt have to be for this PR, but kwargs should be changed to not have a mutable default

Reply:
I can do both in a follow up.

Agreed that this should be `kwargs=None`. The docstring is misleading as well since it says apply takes `**kwargs` so reason enough for a cleanup + full change here.